scripts/utils.js

```diff
@@ -42,3 +42,41 @@ export const [setLibs, getLibs] = (() => {
     }, () => libs,
   ];
 })();
+
+/**
+ * Loads a chat-cta based on metadata
+ */
+export async function loadChatCTA() {
+  const { hostname } = window.location;
+  const branch = hostname.includes('localhost') ? 'http://localhost:3000' : 'https://main--bacom--adobecom.hlx.page';
```

Comment:
What about production and hlx.live?

Reply:
Could you use origin here instead of branch?
```suggestion
  const { origin } = window.location;
```